ScoutSuite/frontend/src/pages/Dashboard/TabsContent/Summary/IssuesCharts/index.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { 
+  PieChart, 
+  Pie,
+  BarChart,
+  Bar,
+  ResponsiveContainer,
+  CartesianGrid,
+  Tooltip,
+  XAxis,
+  YAxis,
+  Legend,
+} from 'recharts';
+import reduce from 'lodash/reduce';
+import get from 'lodash/get';
+import omitBy from 'lodash/omitBy';
+import size from 'lodash/size';
+
+import './style.scss';
+
+
+const propTypes = {
+  services: PropTypes.arrayOf(PropTypes.object).isRequired,
+};
+
+const IssuesCharts = props => {
+  const { services } = props;
+
+  const height = 400;
+  const fillColors = [
+    '#24bc74',
+    '#4099ff',
+    '#ffb44c',
+    '#fc6464',
+    '#c41230',
+  ];
+
+  const issues = reduce(
+    services, 
+    (sum, service) => ({
+      Good: sum.Good + get(service, ['issues', 'Good'], 0),
+      Low: sum.Low + get(service, ['issues', 'Low'], 0),
+      Medium: sum.Medium + get(service, ['issues', 'Medium'], 0),
+      High: sum.High + get(service, ['issues', 'High'], 0),
+      Critical: sum.Critical + get(service, ['issues', 'Critical'], 0),
+    }), 
+    {
+      Critical: 0,
+      High: 0,
+      Medium: 0,
+      Low: 0,
+      Good: 0,
+    },
+  );
+  const pieData = Object.entries(issues).map(([name, sum], i) => ({
+    name: name,
+    value: sum,
+    fill: fillColors[i],
+  }));
+
+  const barData = services.map(service => ({
+    name: service.name,
+    ...omitBy(service.issues, issue => issue === 0),
+  }));
+
+  return (
+    <div className="issues-charts">
+      <div className="pie-chart">
+        <h3>Issues by severity</h3>
+        <ResponsiveContainer 
+          height={height}
+          width="100%"
+        >
+          <PieChart>
```

Comment:
Maybe it could be a good idea to create a functional component for each graph type to simplify the code. Let me know what you think.

Reply:
No, it's okay like this. It's only two graphs, if it was 3+ functional components would be a good idea!